src/main/kotlin/org/wycliffeassociates/otter/jvm/app/widgets/resourcecard/ResourceCard.kt

```diff
@@ -0,0 +1,77 @@
+package org.wycliffeassociates.otter.jvm.app.widgets.resourcecard
+
+import de.jensd.fx.glyphs.materialicons.MaterialIcon
+import de.jensd.fx.glyphs.materialicons.MaterialIconView
+import javafx.beans.property.SimpleBooleanProperty
+import javafx.geometry.Pos
+import javafx.scene.layout.HBox
+import javafx.scene.layout.Priority
+import org.wycliffeassociates.otter.common.data.workbook.Resource
+import org.wycliffeassociates.otter.jvm.app.theme.AppTheme
+import org.wycliffeassociates.otter.jvm.app.widgets.highlightablebutton.highlightablebutton
+import org.wycliffeassociates.otter.jvm.statusindicator.control.StatusIndicator
+import org.wycliffeassociates.otter.jvm.statusindicator.control.statusindicator
+import tornadofx.*
+import tornadofx.FX.Companion.messages
+
+class ResourceCard(private val resource: Resource) : HBox() {
+
+    val isCurrentResource = SimpleBooleanProperty(false)
+
+    init {
+        isFillHeight = false
+        alignment = Pos.CENTER_LEFT
+        maxHeight = 50.0
+
+        vbox {
+            spacing = 3.0
+            hbox {
+                spacing = 3.0
+                add(
+                    statusindicator {
+                        initForResourceCard()
+                        progress = 1.0
+                    }
+                )
+                add(
+                    statusindicator {
+                        initForResourceCard()
+                        progress = 0.0
+                    }
+                )
+            }
+            text(resource.title.text)
+            maxWidth = 150.0
+        }
+
+        region {
+            hgrow = Priority.ALWAYS
+        }
+
+        add(
+            highlightablebutton {
+                primaryColor = AppTheme.colors.appOrange
+                secondaryColor = AppTheme.colors.white
+                isHighlighted = isCurrentResource
+                graphic = MaterialIconView(MaterialIcon.APPS, "25px")
+                maxWidth = 500.0
+                text = messages["viewRecordings"]
+            }
+        )
+    }
+}
+
+fun resourcecard(resource: Resource, init: ResourceCard.() -> Unit = {}): ResourceCard {
+    val rc = ResourceCard(resource)
+    rc.init()
+    return rc
+}
+
+private fun StatusIndicator.initForResourceCard() {
+    prefHeight = 5.0
+    prefWidth = 75.0
+    primaryFill = AppTheme.colors.appOrange
+    accentFill = AppTheme.colors.lightBackground
+    trackFill = AppTheme.colors.defaultBackground
```

Comment:
I think is the same as what joe said in another comment. Don't make the ResouceCard control dependent on the colors coming from the AppTheme. You could set the default values here for the fills with `Color.(whateverColor)` and then expose some ColorProperties that you could expose on the ResourceCard itself. and then bind those to the primaryFill,accentFill and trackFill of the statusIndicator. That way if anyone is using the ResourceCard and wants to change the colors of the statusinidcators they can

Reply:
maybe `progress properties` too ?

Reply:
I'm not sure how we're going to implement progress yet, so maybe a progress property should wait for later? 

I think a color property on ResourceCard for primaryColor/fill is a good idea. I'm not sure about creating properties for accentFill and trackFill yet though since a feature to change those colors on the resourcecard hasn't been established. Thoughts?

Reply:
Yes I actually I think you have a fair point. We don't have feature to change those colors yet, I was just thinking that a situation could arise where we might need a different color, but since that isn't specced out yet we can add that feature to it in the future if needed